.coveragerc

```diff
@@ -9,6 +13,3 @@ exclude_lines =
 partial_branches =
     pragma: no branch
     for .*
-include =
-    asyncssh/*
-    tests/*
```

Comment:
Was there a reason this needed to be removed? Without it, I sometimes see files from other directories popping up in the coverage output when I run things by hand.

Reply:
It was removed because in a prior test, this seems to prevent Windows coverage data from being combined due to the path separator. Also, this option only affects the reporting phase, and using similar option for the execution phase seems better.

Do files from unrelated directories still show up in tests? I have also added `[run] source` ([docs](https://coverage.readthedocs.io/en/latest/source.html#source)) to instruct coverage only measure the `asyncssh` and `tests` package. Any other packages should be ignored.

Reply:
Ah - I missed that you moved these from [report] to [run]. I agree that's a better place.

The problem I saw turned out to be that I had copied the files to a new directory but didn't have the proper coveragerc in place.

Regarding the mixture of UNIX-style and Window-style paths, I've seen that in past experiments trying to merge coverage data between the two, but your last attempt at it here looked good to me.

Are there any remaining issues you're still working on it, or do you think this is good to go?

Reply:
I think it's good for now. I'm also happy to help with any related problems in the future.